src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './YourUploadedCVs.css';
+import { RatingStars } from './RatingStars';
+import { CVList } from './CVList';
+import Button from '../../Button/Button';
+
+export function YourUploadedCVs({ CVsList }) {
+  return (
+    <section className="YourUploadedCVs">
+      <div className="header-yourUploadedCV">
+        <span>
+          <h2>Your Uploaded CVs</h2>
+        </span>
+        <div className="uploadCVBtn">
+          <Button buttonName="Upload new CV" backgroundColor="black" />
+        </div>
+      </div>{' '}
+      {/* CVsList is an array comes as a resault of fetching API GET CVs  */}
+      <div className="CVlistHolder">
+        {CVsList.map((cv, i) => {
+          return i < 3 ? (
+            <ul key={cv.id}>
```

Comment:
You have a list of ul's? Im not sure you use the ul correctly. Remember ul is the wrapper for the list, then each element should be inside an li

Reply:
Done!